CoroutineTemplate/app/src/main/java/co/nimblehq/coroutine/ui/screens/second/SecondViewModel.kt

```diff
@@ -1,5 +0,0 @@
-package co.nimblehq.coroutine.ui.screens.second
-
-import co.nimblehq.coroutine.ui.base.BaseViewModel
-
-class SecondViewModel : BaseViewModel()
```

Comment:
pls follow the rule that every fragment or activity should contains at least 1 view model